test.js

```diff
@@ -0,0 +1,276 @@
+
+        const expected = `• 1st level\n  • 2nd level\n    • 3rd level\n      • 4th level\n\n\n    • 3rd level back\n\n\n  • 2nd level back\n\n  • 2nd level back again\n\n\n• 1st level back\n\n`;
+        expect(slackify(input)).toBe(expected);
+      });
+      it("should handle non-nested <ol> tags appropriately", () => {
+        const input =
+          '<ol class="ghq-card-content__numbered-list" data-ghq-card-content-type="NUMBERED_LIST" start="1"><li class="ghq-card-content__numbered-list-item" data-ghq-card-content-type="NUMBERED_LIST_ITEM">test 1</li><li class="ghq-card-content__numbered-list-item" data-ghq-card-content-type="NUMBERED_LIST_ITEM">test 2</li><li class="ghq-card-content__numbered-list-item" data-ghq-card-content-type="NUMBERED_LIST_ITEM">test 3</li></ol><p class="ghq-card-content__paragraph ghq-is-empty" data-ghq-card-content-type="paragraph"></p>';
+        const expected = "1. test 1\n2. test 2\n3. test 3\n\n";
+        expect(slackify(input)).toBe(expected);
+      });
+      it("should handle nested <ol> tags appropriately", () => {
+        const input =
+          '<ol start="1" class="ghq-card-content__numbered-list" data-ghq-card-content-type="NUMBERED_LIST"><li class="ghq-card-content__numbered-list-item" data-ghq-card-content-type="NUMBERED_LIST_ITEM"><p class="ghq-card-content__paragraph" data-ghq-card-content-type="paragraph">hi</p><ol start="1" class="ghq-card-content__numbered-list" data-ghq-card-content-type="NUMBERED_LIST"><li class="ghq-card-content__numbered-list-item" data-ghq-card-content-type="NUMBERED_LIST_ITEM"><p class="ghq-card-content__paragraph" data-ghq-card-content-type="paragraph">hello</p><ol start="1" class="ghq-card-content__numbered-list" data-ghq-card-content-type="NUMBERED_LIST"><li class="ghq-card-content__numbered-list-item" data-ghq-card-content-type="NUMBERED_LIST_ITEM"><p class="ghq-card-content__paragraph" data-ghq-card-content-type="paragraph">TEST</p></li><li class="ghq-card-content__numbered-list-item" data-ghq-card-content-type="NUMBERED_LIST_ITEM"><p class="ghq-card-content__paragraph" data-ghq-card-content-type="paragraph">test1</p></li></ol></li><li class="ghq-card-content__numbered-list-item" data-ghq-card-content-type="NUMBERED_LIST_ITEM"><p class="ghq-card-content__paragraph" data-ghq-card-content-type="paragraph">hellpo 1</p></li></ol></li><li class="ghq-card-content__numbered-list-item" data-ghq-card-content-type="NUMBERED_LIST_ITEM"><p class="ghq-card-content__paragraph" data-ghq-card-content-type="paragraph">basd</p></li></ol>';
+        const expected = "";
+
+        // expect(slackify(input)).toBe(expected);
```

Comment:
since ordered lists technically aren't correct yet I wasn't sure whether I should just use what we have as the correct output. I left this here for now because the input at least will be helpful if someone tackles it later, but open to suggestions here